_includes/c11-cpp11-and-beyond-and-toolchains.md

```diff
@@ -145,7 +145,7 @@ matrix:
       env:
         - MATRIX_EVAL="CC=clang-4.0 && CXX=clang++-4.0"
 
-    # works on Trusty
+    # Currently doesn't work on Trusty
```

Comment:
This does not work still if I can see, Will restart a build later to double check.